src/transformers/modeling_lxmert.py

```diff
@@ -0,0 +1,975 @@
+         token_type_ids (:obj:`torch.LongTensor` of shape :obj:`{0}`, `optional`, defaults to :obj:`None`):
+             Segment token indices to indicate first and second portions of the inputs.
+             Indices are selected in ``[0, 1]``: ``0`` corresponds to a `sentence A` token, ``1``
+             corresponds to a `sentence B` token
+
+             `What are token type IDs? <../glossary.html#token-type-ids>`_
+
+
+        visual_feats: (:obj: `Tuple[torch.FloatTensor, torch.FloatTensor]`):
+            the first item in the tuple represents the actual visual features
+            (ROI pooled object features from bounding boxes using a faster-RCNN model) of shape ՝(batch_size, num_visual_features, visual_feat_dim)՝
+            and where the second item represents the normalized bounding boxes on a scale of 0~1  of shape ՝(batch_size, 4)՝.
+            These are currently not provided by the transformers library
+        visual_attention_mask (:obj:`torch.FloatTensor` of shape :obj:`{0}`, `optional`, defaults to :obj:`None`):
+             Mask to avoid performing attention on padding token indices.
+             Mask values selected in ``[0, 1]``:
+             ``1`` for tokens that are NOT MASKED, ``0`` for MASKED tokens.
+        output_v_attentions: (:obj:`bool`, `optional`, defaults to :obj:`False`):
+             If set to ``True``, the attentions tensors of all attention layers for the visual encoder are returned. See ``attentions`` under returned tensors for more detail.
+        output_l_attentions: (:obj:`bool`, `optional`, defaults to :obj:`False`):
+             If set to ``True``, the attentions tensors of all attention layers for the language encoder are returned. See ``attentions`` under returned tensors for more detail.
+        output_x_attentions: (:obj:`bool`, `optional`, defaults to :obj:`False`):
+             If set to ``True``, the attentions tensors of all attention layers for the cross modality encoder are returned. See ``attentions`` under returned tensors for more detail.
+         output_v_hidden_states (:obj:`bool`, `optional`, defaults to :obj:`False`):
+             If set to ``True``, the visual hidden states of all layers from the cross modality encoder are returned where the visual features are used as the input and the language features are used as the context. See ``hidden_states`` under returned tensors for more detail.
+         output_l_hidden_states (:obj:`bool`, `optional`, defaults to :obj:`False`):
+             If set to ``True``, the language hidden states of all layers from the cross modality encoder are returned where the visual features are used as the context. See ``hidden_states`` under returned tensors for more detail.
+        return_dict (:obj:`bool`, `optional`, defaults to :obj:`None`):
+            If set to ``True``, the model will return a :class:`~transformers.file_utils.LxmertModelOutput` instead of a
+            plain tuple.
```

Comment:
Great that you added the `return_dict`! I think it's lacking a space for indentation?